### ISSUE TITLE
TreeView control has no highlighting of selected node first time if the control has first TabIndex on the form

### ISSUE THREAD

**Issue by lisoenot** [dotnet/winforms#6559]
* .NET Core Version: 
7.0.0-alpha.1.21528.1

* Have you experienced this same bug with .NET Framework?: 
Yes

* More info:
This problem was found in #6432
You can find more details there.

**Problem description:**
If a TreeView control has first TabIndex on the form, selected node has no highlighting first time.

![BrokenFocusOnTreeView](https://user-images.githubusercontent.com/58004471/147512386-b429de63-9bda-4498-bc64-5bd25076beaa.gif)

**Expected behavior:**
Selected node should have focus all times whatever TabIndex of control.

![FocusOnTreeView](https://user-images.githubusercontent.com/58004471/147512415-1f71ba67-4201-43e7-b7db-1c6381e94b05.gif)

**Minimal repro:**
1. Create a WinForms core project
1. Create TreeView with nodes.
1. Then create Button.
1. Make sure tree has TabIndex equals 0 and button has TabIndex equals 1. If no change it.
1. Launch Inspect or Narrator.
1. Build and run the app.
1. Make sure when the form finishes loading the selected node is not highlighted.

**Comment by Ashley-Li** [comment]
Verified this issue from main branch with winforms repo, it was fixed. Now TreeView control has highlighting of selected node first time if the control has first TabIndex on the form.

https://github.com/dotnet/winforms/assets/56664234/1a34876a-d25a-4bc4-8a83-90d694e80732
